test/e2e/config/ironic.yaml

```diff
@@ -32,6 +32,7 @@ variables:
   IMAGE_URL: "http://192.168.222.1/cirros-0.6.2-x86_64-disk.img"
   IMAGE_CHECKSUM: "c8fc807773e5354afe61636071771906"
   CERT_MANAGER_VERSION: "v1.13.0"
+  RUN_UPGRADE_TESTS: "true"
```

Comment:
I don't think we should control what tests to run in this way. We can use `GINKGO_SKIP` to skip specific tests if needed.

Reply:
Yes. But my point here is that we should never run upgrade tests with fixture config. We can always add GINKGO_SKIP if we want to.

Reply:
But my point is why should we not? As the test is now, I don't see why it would not work with fixture. In fact I think fixture is the only place it can work currently without Ironic